guides/common/modules/proc_restricting-custom-repositories-to-a-specific-os-version-or-architecture.adoc

```diff
@@ -0,0 +1,20 @@
+[id="Restricting_Custom_Repositories_to_a_Specific_OS_Version_{context}"]
+= Restricting Custom Repositories to a Specific Operating System or Architecture in {Project}
+
+You can configure {Project} to make custom repositories available only on hosts with a specific operating system version or architecture.
+For example, you can restrict custom repositories only to {RHEL}{nbsp}9 hosts.
+
+ifdef::satellite[]
```

Comment:
IMHO this also applies to Katello + orcharhino _if_ you manage both content from Red Hat and other "custom" content.

Because we don't have an appropriate attribute for this yet, leave as is.